### PR TITLE
fix: do not assume default sqs queue url

### PR DESCRIPTION
**Pull request by stephhuynh18**
It is very weird that in kubernetes the default config does not populate sqsQueueUrl

### CODE REVIEW

**Review by smrz2001 — APPROVED** [review]
🥳

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -160,7 +160,7 @@ export class AnchorService {
   ) {
     this.merkleDepthLimit = config.merkleDepthLimit
     this.useSmartContractAnchors = config.useSmartContractAnchors
-    this.useQueueBatches = config.queue.sqsQueueUrl !== ''
+    this.useQueueBatches = Boolean(config.queue.sqsQueueUrl)
```
Comment (3benbox) [inline src/services/anchor-service.ts:163]: I think this is due to javascript's truthyness vs Go?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Either way, LGTM!

**Review by 3benbox — APPROVED** [review]
LGTM

File: src/services/anchor-service.ts (hunk 1 above)
Comment (stephhuynh18) [reply src/services/anchor-service.ts:163]: I'm actually assuming that if the sqsQueueUrl isn't set, the default config will fill it in with an empty string. Thats why the issues you and Mohsin are seeing are so weird....

**Pull request merged**